Languages/English/Keyed/AllKeyed_Help.xml

```diff
@@ -106,57 +106,57 @@ Section 1.1 - Physical properties
 
   <!--================================ Cultivators ==============================-->
   <!-- Drone Cultivators -->
-  <PRF_DroneCultivator_I_HelpText>A first generation agricultural drone specifically tasked to tend to plants. It will not haul the final product to storage.</PRF_DroneCultivator_I_HelpText>
+  <PRF_DroneCultivator_I_HelpText>A first-generation agricultural drone specifically tasked to tend to plants. It will not haul the final product to storage.</PRF_DroneCultivator_I_HelpText>
   <PRF_DroneCultivator_II_sun_HelpText>An improved version of the initial drone cultivator variant. This second-generation drone cultivator is capable of faster tending of crops, hauling, and cutting. This variant has a sunlamp attached to it, allowing crops to grow even in the dark.</PRF_DroneCultivator_II_sun_HelpText>
   <PRF_DroneCultivator_II_HelpText>An improved version of the initial drone cultivator variant. This second-generation drone cultivator is capable of faster tending of crops, hauling, and cutting.</PRF_DroneCultivator_II_HelpText>
-  <PRF_DroneCultivator_III_HelpText>A third-generation drone cultivator and of the latest design. It will work with superior agricultural skills and stations additional drones compared to its predecessor. It will cut, grow, harvest, and haul any items to a stockpile if available within range.</PRF_DroneCultivator_III_HelpText>
+  <PRF_DroneCultivator_III_HelpText>A third-generation drone cultivator, and of the latest design. It will work with superior agricultural skills and station additional drones compared to its predecessor. It will cut, grow, harvest, and haul any items to a stockpile if available within the range.</PRF_DroneCultivator_III_HelpText>

```

Comment:
maybe `holds` would be a better term

Reply:
You mean here? "agricultural skills and station additional drones" -> "agricultural skills and hold additional drones"

Reply:
yes